cmd/rot13.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/getlantern/rot13"
+)
+
+func die(format string, args ...interface{}) {
+	fmt.Fprintf(os.Stderr, format, args...)
+}
+
+func main() {
+	var err error
+
+	inFile := flag.String("input", "", "Input file")
+	outFile := flag.String("output", "", "Output file")
+	encode := flag.Bool("encode", false, "Encode the input string (default is to decode)")
+	flag.Parse()
+
+	var in io.Reader
+	if *inFile == "-" || *inFile == "" {

```

Comment:
Would be nice to document this feature in the usage string.
Also, would be nice to have a usage string (i.e., `./rot13 -h`)

Reply:
I added that to the usage string. Passing `-h` does already work, though! (See updated README in a minute for an example.)